edit/persistentvolume/index.vue

```diff
@@ -44,13 +44,15 @@ export default {
   async fetch() {
     const storageClasses = await this.$store.dispatch('cluster/findAll', { type: STORAGE_CLASS });
     const pvcPromise = this.$store.dispatch('cluster/findAll', { type: PVC });
+    const featureFlagPromise = this.$store.dispatch('management/find', { type: MANAGEMENT.FEATURE, id: 'unsupported-storage-drivers' });
 
     this.storageClassOptions = storageClasses.map(s => ({
       label: s.name,
       value: s.name
     }));
     this.storageClassOptions.unshift(this.NONE_OPTION);
     await pvcPromise;
+    this.showUnsupportedStorage = (await featureFlagPromise).enabled;

```

Comment:
And should handle the flag not being found (they can be deleted...)

Reply:
It looks like they actually can't but I'll account for this case anyway.

![image](https://user-images.githubusercontent.com/55104481/111231825-24e5ba80-85a7-11eb-9e27-f3e91d548efb.png)

Reply:
Ok not "deleted" by the user, but can cease to exist in the version of code you're pointed at.